applications/compareRestart.hpp

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
Also (thought this was not said already, sorry if I repeat stuff) the header and class implementation should then go in one of the "usual" opm directories, for example opm/output/eclipse, the header marked for installation in the CMakeList_files.cmake, and the program using it should include it with full path (`#include <opm/...>`).